roles/satellite/tasks/define-auth-ldap.yml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
I have trouble understanding what you want to do with this task here.

The current workflow is:
* one playbook to defined Orgas, domains, subnets, OSes, provisionning templates, etc
* one playbook to create Content Views, lifecycle environment, etc
* some playbooks to manage lifecycle of ContentViews/OSes (change, publish, promote, etc)

What do you want to do with these tasks ?

Reply:
ok.... Maybe  "authentication setup" can be included in the playbook define-infra ?

Reply:
At least organizations need to be defined before setting up ldap authentication since they are referenced in the field `organizations`.

I think putting the ldap authentication in the initial-define-infra yml tasks is fine.

Reply:
Note: you also want to use https://github.com/theforeman/foreman-ansible-modules/blob/develop/plugins/modules/external_usergroup.py to setup roles for different groups coming from the LDAP directory.